interface/billing/billing_report.php

```diff
@@ -1111,8 +1111,8 @@ class='link_submit'  ><?php echo '['.xlt('End Of Day Report').']' ?></a>
           $rhtml2 .= text(oeFormatSDFT(strtotime($date)));
           $rhtml2 .= "</span></td>\n";
           if ($iter['id'] && $iter['authorized'] != 1) {
-            $rhtml2 .= "<td><span class=alert>".xlt("Note: This code was not entered by an authorized user. Only authorized codes may be uploaded to the Open Medical Billing Network for processing. If you wish to upload these codes, please select an authorized user here.")."</span></td>\n";
-          }else{
+		  $rhtml2 .= "<td><span class=alert>".xlt("Note: This copay was entered against unauthorized billing.  Please review status..")."</span></td>\n";
```

Comment:
Just trying to sort out the logic here. So does this basically flags if the charge was not authorized? If so, can't we just have an errors stating the code/copay was not authorized or something like that.